tunnel/default.go

```diff
@@ -139,16 +138,17 @@ func (t *tun) monitor() {
 			return
 		case <-reconnect.C:
 			for _, node := range t.options.Nodes {
-				t.Lock()
```

Comment:
This should still be here as its a race condition to read from the map without a lock

Reply:
Oops. Beats me how and why I swapped the order here 🤦‍♂️